index.html

```diff
@@ -198,10 +221,10 @@ <h2>Contact me</h2>
                 <input type="email" id="email" placeholder="Enter your Email"  minlength="2" required >
                 <textarea name="textarea" id="message" placeholder="Write your Message here" required  minlength="2" maxlength="500"></textarea>
                     <button class="btn-contact" type="submit" required>Get in touch</button>
```

Comment:
- [ ] [OPTIONAL] On hover state, I kindly suggest that you add a border-color of white around your button to make sure your button color does not merge with your contact form background-color